tools/python/util/vcpkg_helpers.py

```diff
@@ -367,66 +411,81 @@ def generate_vcpkg_triplets_for_emscripten(build_dir: str, emscripten_root: str)
         build_dir (str): The directory to save the generated triplet files.
         emscripten_root (str): The root path of Emscripten.
     """
-    for enable_rtti in [True, False]:
-        for enable_asan in [True, False]:
-            for target_abi in ["wasm32", "wasm64"]:
-                folder_name_parts = []
-                if enable_asan:
-                    folder_name_parts.append("asan")
-                folder_name = "default" if len(folder_name_parts) == 0 else "_".join(folder_name_parts)
-                os_name = "emscripten"
-                folder_name_parts = []
-                if enable_asan:
-                    folder_name_parts.append("asan")
-                if not enable_rtti:
-                    folder_name_parts.append("nortti")
-                folder_name = "default" if len(folder_name_parts) == 0 else "_".join(folder_name_parts)
-                file_name = f"{target_abi}-{os_name}.cmake"
-                dest_path = Path(build_dir) / folder_name / file_name
-                os.makedirs(dest_path.parent, exist_ok=True)
-                with open(dest_path, "w", encoding="utf-8") as f:
-                    add_copyright_header(f)
-                    f.write(r"""
+    enable_minimal_build = False  # Assume False for Emscripten unless specified otherwise
```

Comment:
Exception can only be disabled if and only if exception is false.

Reply:
TODO: only produces two triplet files: wasm32 and wasm64. Pass in all the other parameters.